Dockerfile

```diff
@@ -34,6 +38,8 @@ RUN git checkout $MENDER_VERSION
 # Copy the debian recipe(s)
 COPY debian-master debian-master
 COPY debian-2.1.x debian-2.1.x
+COPY debian-2.2.x debian-2.3.x
+COPY debian-2.3.x debian-2.2.x
```

Comment:
The destination folder is opposite of the source folder. Typo?

Reply:
Yes, typo :) Thanks!

Ironically one is a symlink to the other one, so technically doesn't matter. I'll fix.